task01.rb

```diff
@@ -2,11 +2,11 @@
 
 def task01(str)
   str.gsub!(/ /, "")
-  if str.scan(/[<>]=?\d+/)[0].length == str.length 
+  if str.scan(/[<>]?=?\d+/)[0].length == str.length 
     true
+  elsif str.scan(/\d+/)[0].length == str.length 
+    false
```

Comment:
@MaraEfimova Is this the only case when the program should return `false`? Can we say that there are no other cases when the program should return `false`? Do we really need this additional `if` here?